packages/rainbowkit/src/wallets/walletConnectors/tokenaryWallet/tokenaryWallet.ts

```diff
@@ -0,0 +1,34 @@
+import type { InjectedConnectorOptions } from '@wagmi/core/connectors/injected';
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { isAndroid } from '../../../utils/isMobile';
+import { Wallet } from '../../Wallet';
+
+export interface TokenaryWalletOptions {
+  chains: Chain[];
+}
+
+export const tokenary = ({
+  chains,
+  ...options
+}: TokenaryWalletOptions & InjectedConnectorOptions): Wallet => ({
+  id: 'tokenary',
+  name: 'Tokenary',
+  iconUrl: async () => (await import('./tokenary.svg')).default,
+  iconBackground: '#000000',
+  installed:
+    typeof window !== 'undefined' &&
+    typeof window.ethereum !== 'undefined' &&
+    window.ethereum.isTokenary,
+  hidden: () => isAndroid(),
```

Comment:
@grachyov Is Tokenary unsupported on Android? Typically we'd still leave the wallet connector visible on the dApp even on unsupported platforms. The exceptions we have thus far are for the `safeWallet` and `injectedWallet` connectors where they are only available in edge cases like the Safe Browser or when an injected wallet connector is not available and the user would need a button to connect their wallet.

Reply:
@DanielSinclair yeah, tokenary is not supported on android. only ios and macos safari.

Reply:
@DanielSinclair what should i do to these deployment errors?